libs/ledgerjs/packages/types-live/src/feature.ts

```diff
@@ -36,9 +31,9 @@ export type Feature<T = unknown> = {
   mobile_version?: string;
 
   /**
-   * Whether the remote value of `enabled` was overriden due to `mobile_version`
+   *  Whether the remote value of `enabled` was overriden due to `desktop_version` or `mobile_version`
    */
-  enabledOverriddenForCurrentMobileVersion?: boolean;
+  enabledOverriddenForCurrentVersion?: boolean;
```

Comment:
yes, we can. I just merge the current "enabledOverriddenForCurrentMobileVersion"(only used in LLM) and "enabledOverriddenForCurrentDesktopVersion"(only used in LLD) into one variable.
In "libs/live-config/src/featureFlags/firebaseFeatureFlags.ts"
we have this code now:

```
  const featureSpecificVersion: string | undefined = (() => {
    switch (platform) {
      case "desktop":
        return feature.desktop_version;
      case "ios":
      case "android":
        return feature.mobile_version;
      default:
        return undefined;
    }
  })();
  if (
    featureSpecificVersion &&
    !semver.satisfies(LiveConfig.getInstance().appVersion, featureSpecificVersion, {
      includePrerelease: true,
    })
  ) {
    return {
      enabledOverriddenForCurrentVersion: true,
      ...feature,
      enabled: false,
    };

```

enabledOverriddenForCurrentVersion takes account into consideration both feature.desktop_version and feature.mobile_version because featureSpecificVersion is calculated based on feature.desktop_version and feature.mobile_version.